python.md

```diff
@@ -758,3 +758,55 @@ def test_some_longwinded_process(support_client, factory):
 ```
 
 You get the idea.
+
+
+## Documentation
+
+### READMEs
```

Comment:
```suggestion
### Domain package READMEs
```

This section probably belongs in the "General Python" section, next to the existing note on docstrings.